core/src/toga/window.py

```diff
@@ -690,6 +691,7 @@ def question_dialog(
             "question_dialog(...) has been deprecated; "
             "use dialog(toga.QuestionDialog(...))",
             DeprecationWarning,
+            stacklevel=2,
```

Comment:
Continued theme - a pre-existing problem, but multi-line strings should be in brackets. Rinse and repeat for the rest of the PR - I won't call them all out individually; anywhere that you're adding `stacklevel=2` is a point to check.